drv/stm32h7-spi-server/src/main.rs

```diff
@@ -197,16 +198,19 @@ impl InOrderSpiImpl for ServerImpl {
         // Reject out-of-range devices.
         let device = CONFIG.devices.get(devidx).ok_or(SpiError::BadDevice)?;
 
-        // If we're asserting CS, we want to *reset* the pin. If
-        // we're not, we want to *set* it. Because CS is active low.
-        let pin_mask = device.cs.pin_mask;
-        self.sys
-            .gpio_set_reset(
-                device.cs.port,
-                if cs_asserted { 0 } else { pin_mask },
-                if cs_asserted { pin_mask } else { 0 },
-            )
-            .unwrap();
+        for pin in device.cs {
+            // If we're asserting CS, we want to *reset* the pin. If
+            // we're not, we want to *set* it. Because CS is active low.
+            let pin_mask = pin.pin_mask;
+            self.sys
```

Comment:
What about this?
```rust
if cs_asserted {
    self.sys.gpio_reset(pin).unwrap();
} else {
    self.sys.gpio_set(pin).unwrap();
}
```

Reply:
Definitely prefer it -- fixing, and thank you!